build/restore.ps1

```diff
@@ -1,7 +1,14 @@
 $root = $PSScriptRoot
+$packages = "$root\..\packages"
+
 $nuget = & "$root\Get-Nuget.ps1"
 
-& $nuget restore $root\..\PortabilityTools.sln
+if (!(Test-Path $packages))
+{
+    New-Item $packages -ItemType Directory
```

Comment:
Ah sorry, mkdir is a command itself since XP. The alias of New-Item is `ni`.
mkdir is a DOS command, which means it was there for 30+ years :smile:

PowerShell let us add `-ErrorAction Ignore` to any command, which has the same effect as `-p` switch of mkdir command in Unix shells.